src/Workleap.DomainEventPropagation.Publishing.Tests/DomainEventWrapperTests.cs

```diff
@@ -35,26 +30,6 @@ public void GivenEventGridEvent_WhenSetMetadata_ThenMetadataIsSet()
         Assert.Equal("someValue", value);
     }
 
-    [Fact]
-    public void GivenCloudEvent_WhenSetMetadata_ThenExceptionIsThrown()
```

Comment:
Don't think those should just be deleted but instead have their logic adjusted to the proper behavior. We don't throw anymore, but we might want to do the same validations as we do with EventGridEvents

Reply:
good point, added similar tests for cloudevents